gradlePlugin/fhir-dependency/src/main/kotlin/care/data4life/hl7/fhir/dependency/Dependency.kt

```diff
@@ -0,0 +1,175 @@
+ * applications/third-party applications shall require the conclusion of a license agreement
+ * between you and D4L.
+ *
+ * If you are interested in licensing the SDK for your own applications/third-party
+ * applications and/or if you’d like to contribute to the development of the SDK, please
+ * contact D4L by email to help@data4life.care.
+ */
+
+package care.data4life.hl7.fhir.dependency
+
+object Dependency {
+
+    val kotlin = Kotlin
+    val multiplatform = Multiplatform
+    val multiplatformTest = MultiplatformTest
+    val jvm = Jvm
+    val jvmTest = JvmTest
+    val android = Android
+    val androidTest = AndroidTest
+
+    object Kotlin {
+        const val stdLib = "org.jetbrains.kotlin:kotlin-stdlib:${Version.kotlin}"
+        const val reflect = "org.jetbrains.kotlin:kotlin-reflect:${Version.kotlin}"
+        const val serialization = "org.jetbrains.kotlin:kotlin-serialization:${Version.kotlin}"
+    }
+
+    object Multiplatform {
+
+        val kotlin = Kotlin
+        val d4l = D4L
+        val koin = Koin
+
+        object Kotlin {
+
+            val stdLib = StdLib
+            val coroutines = Coroutines
+            val serialization = Serialization
+
+            object StdLib {
+                const val common = "org.jetbrains.kotlin:kotlin-stdlib-common:${Version.kotlin}"
+                const val jdk = "org.jetbrains.kotlin:kotlin-stdlib:${Version.kotlin}"
+                const val jdk8 = "org.jetbrains.kotlin:kotlin-stdlib-jdk8:${Version.kotlin}"
+                const val js = "org.jetbrains.kotlin:kotlin-stdlib:${Version.kotlin}"
+                const val native = "org.jetbrains.kotlin:kotlin-stdlib:${Version.kotlin}"
+                const val android = "org.jetbrains.kotlin:kotlin-stdlib:${Version.kotlin}"
+            }
+
+            object Coroutines {
+                const val common = "org.jetbrains.kotlinx:kotlinx-coroutines-core:${Version.multiplatform.kotlin.coroutines}"
+                const val android = "org.jetbrains.kotlinx:kotlinx-coroutines-android:${Version.multiplatform.kotlin.coroutines}"
+                const val test = "org.jetbrains.kotlinx:kotlinx-coroutines-test:${Version.multiplatform.kotlin.coroutines}"
+            }
+
+            object Serialization {
+                const val json = "org.jetbrains.kotlinx:kotlinx-serialization-json:${Version.multiplatform.kotlin.serialization}"
+            }
+
+            const val dateTime = "org.jetbrains.kotlinx:kotlinx-datetime:${Version.multiplatform.kotlin.dateTime}"
+        }
+
+        object D4L {
+
+            val util = Util
+
+            object Util {
+                const val common = "care.data4life.hc-util-sdk-kmp:util:${Version.multiplatform.d4l.utilSdk}"
+                const val android = "care.data4life.hc-util-sdk-kmp:util-android:${Version.multiplatform.d4l.utilSdk}"
+                const val jvm = "care.data4life.hc-util-sdk-kmp:util-jvm:${Version.multiplatform.d4l.utilSdk}"
```

Comment:
I think we only can use common now, since we bump there to Kotlin 1.4.32, but it so no blocker for me.

Reply:
yeah, but I had some issues and no time to closely look if they are comming from removing this ore from somewhere else